Integrations/Cybereason/Cybereason.yml

```diff
@@ -552,8 +728,10 @@ script:
     - contextPath: Cybereason.User.Organization
       description: User organization
       type: string
-    description: Query users as part of investigation
+  dockerimage: demisto/python
```

Comment:
why did you add this ??

Reply:
I'm getting an issue in Circle which I'm not sure why it's happening. Thought it was trying to use docker with python 3. Turns out it wasn't the cause